src/main/java/com/Studentlifr/GratitudeBoard/dto/requests/MessageReqDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.Studentlifr.GratitudeBoard.dto.requests;
+
+import com.Studentlifr.GratitudeBoard.dao.UserDAO;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+public class MessageReqDTO {
+
+    @NotEmpty
```

Comment:
Add a custom error message

Reply:
Can you please elaborate on what custom error message is required?